app/errors.py

```diff
@@ -0,0 +1,19 @@
+class VaccineError(Exception):
+    ...
+    # def __str__(self):
+    #     return "All friends should be vaccinated"
+
+
+class NotVaccinatedError(VaccineError):
+    ...
+
+
+class OutdatedVaccineError(VaccineError):
+    ...
+
+
+class NotWearingMaskError(Exception):
+    masks_to_buy = 0
+
+    def __str__(self) -> str:
```

Comment:
This message doesn't belong here - it should be returned after all friends are processed